cmd/cli/app.go

```diff
@@ -65,7 +65,6 @@ func newApp() *cli.App {
 			Out: os.Stderr,
 		},
 	)
-	app.Before = common.SetupConsoleLogger
```

Comment:
Imho it would be better to loop app.Commands and set it there than for each command adding it manually

Reply:
much better indeed